3rdparty/CMakeLists.txt

```diff
@@ -117,31 +117,44 @@ if (IRR_BUILD_CEGUI)
 
 	if (MSVC)
 		set(CEGUI_LIBS
-			"${CEGUI_INSTALL_PATH}/lib/CEGUIBase-0_Static.lib "
-			"${CEGUI_INSTALL_PATH}/lib/CEGUICommonDialogs-0_Static.lib "
-			"${CEGUI_INSTALL_PATH}/lib/CEGUICoreWindowRendererSet_Static.lib "
-			"${CEGUI_INSTALL_PATH}/lib/CEGUIExpatParser_Static.lib "
-			"${CEGUI_INSTALL_PATH}/lib/CEGUIOpenGLRenderer-0_Static.lib "
-			"${CEGUI_DEPS_INSTALL_PATH}/lib/Expat.lib "
-			"${CEGUI_DEPS_INSTALL_PATH}/lib/Freetype.lib "
+			${CEGUI_INSTALL_PATH}/lib/CEGUIOpenGLRenderer-0_Static.lib
```

Comment:
not quoting anymore?

Reply:
Yes, not sure why it doesn't work before although I tried different formats including non-quotes and quotes. But right now Windows and Linux are working fine anyways.